apps/prairielearn/src/pages/instructorRequestCourse/instructorRequestCourse.html.ts

```diff
@@ -207,6 +219,35 @@ function CourseNewRequestCard({ csrfToken }: { csrfToken: string }): HtmlValue {
               not need to provide this if you would like to use the online web editor.
             </small>
           </div>
+          <div class="form-group">
+            <label id="cr-referral-source-label">How did you hear about PrairieLearn?</label>
+            <select
+              class="custom-select"
+              name="cr-referral-source"
+              id="cr-referral-source"
+              aria-labelledby="cr-referral-source-label"
+              required
+            >
+              <option value="" disabled selected></option>
+              <option value="I've used PrairieLearn before">I've used PrairieLearn before</option>
+              <option value="Word of mouth">Word of mouth</option>
+              <option value="Web search">Web search</option>
+              <option value="Conference or workshop">Conference or workshop</option>
+              <option value="Publication">Publication</option>
+              <option value="other">Other...</option>
+            </select>
+            <input
+              type="text"
+              class="form-control mt-2 d-none"
+              name="cr-referral-source-other"
+              id="cr-referral-source-other"
+              aria-labelledby="cr-referral-source-label"
+            />
+            <small class="form-text text-muted">
+              This information helps us understand how people find out about PrairieLearn. Thank you
+              for sharing!
```

Comment:
The rest of our inputs have explanatory text, figured it didn't hurt to add some here. Bikeshed away on exactly what this says!